api/src/main/java/org/openmrs/module/insuranceclaims/api/client/ClientConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.openmrs.module.insuranceclaims.api.client;
+
+public final class ClientConstants {
+
+    public static final String API_LOGIN_PROPERTY = "insuranceclaims.externalApiLogin";
+    public static final String API_PASSWORD_PROPERTY = "insuranceclaims.externalApiPassword";
+    public static final String USER_AGENT = "user-agent";
+    public static final String CLIENT_HELPER_USER_AGENT = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36";
```

Comment:
it is really weird that you need to use it, please at least write comment in code why you added it